engine/src/main/java/org/terasology/rendering/cameras/Camera.java

```diff
@@ -204,6 +206,22 @@ public Vector3f getViewingDirection() {
         return viewingDirection;
     }
 
+    /**
+     * Get the gaze direction.
+     * @return the gaze direction, a quaternion.
+     */
+    public Quat4f getGazeDirection() {
```

Comment:
Hmmm.... I find that the term "gazing" implies an agent, i.e. a human, an animal or an autonomous robot.

Cameras however, even though right now are mostly related to the player, could be attached to inanimate
objects, such as in-game security cameras. 

In this context I wouldn't use the term "gaze" in here. If you really need a gaze direction I'd retrieve a vector from whatever dictates the camera orientation and in your code I'd then name it gazeDirection.

But even before that, why isn't getViewingDirection() enough?